tool/teleport/common/teleport.go

```diff
@@ -89,6 +89,7 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	forward := app.Command(teleport.ForwardSubCommand, "Used internally by Teleport to re-exec itself to port forward.").Hidden()
 	checkHomeDir := app.Command(teleport.CheckHomeDirSubCommand, "Used internally by Teleport to re-exec itself to check access to a directory.").Hidden()
 	park := app.Command(teleport.ParkSubCommand, "Used internally by Teleport to re-exec itself to do nothing.").Hidden()
+	waitNoResolve := app.Command(teleport.WaitNoResolveSubCommand, "Used internally by Teleport to wait until the TELEPORT_WAIT_NO_RESOLVE_DOMAIN stops resolving IP addresses.").Hidden()
```

Comment:
Nit: Use the constant you defined instead of hardcoding `TELEPORT_WAIT_NO_RESOLVE_DOMAIN` name here.

Reply:
Switeched to a CLI parameter